requirements.txt

```diff
@@ -4,5 +4,5 @@ attrs==17.4.0
 google-api-core==0.1.4
 google-auth==1.3.0
 google-cloud-pubsub==0.30.1
-gordon-janitor==0.0.1.dev1
+gordon-janitor==0.0.1.dev3
```

Comment:
This might be a dumb thing to ask, but why bump the version from 1 to 3?

Reply:
Because `dev2` failed to upload with all those travis issues last week (\*sighs heavily\*).